src/rubintv/handlers/external/endpoints.py

```diff
@@ -191,13 +202,19 @@ async def update_todays_table(request: web.Request) -> web.Response:
             bucket, camera, recent_events, the_date
         )
         events = flatten_events_dict_into_list(camera, events_dict)
-        grid_columns = build_grid_columns_css(camera.channels)
+
+        metadata_json = "{}"
+        metadata_url = get_metadata_url(bucket.name, camera.slug, date_str)
+        metadata_res = requests.get(metadata_url)
+        if metadata_res.status_code == 200:
```

Comment:
What happens if the return code isn't 200?  Obviously, `metadata_json` is empty, but is that OK for the downstream processing?

Reply:
Yup, I've checked, that works as intended.